code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -176,6 +176,8 @@
 
 	for(var/t in adjacent_turfs)
 		var/turf/open/enemy_tile = t
+		if(!enemy_tile || istype(enemy_tile))
+			return
```

Comment:
This breaks atmos processing and doesn't even fix the bug, the real issue is that something is skipping over `ChangeTurf()` and changing an open turf to a closed one without calling `ImmediateCalculateAdjacentTurfs()`.

Reply:
this all runtimes i see when spawn shuttles and some ruins

Reply:
problem where we load template, before it initialised, adjastment turfs try to interract with unititalised turf.